spec/forms/new_phone_form_spec.rb

```diff
@@ -151,6 +151,22 @@
       expect(result.errors).to be_empty
     end
 
+    context 'when the user has already added the number' do
```

Comment:
if we include the normalization code above, can we also add tests that check for numbers that are the same after normalization (saving one to the DB in a `(XXX) 123-1234` format, etc)